File: Programmers_algo/01_Level1/119_nextSqaure/question.py
# https://programmers.co.kr/learn/challenge_codes/119

def nextSqure(n):
    sr = n**(1/2)
    return int(sr)**2 == n and (sr+1)**2 or 'no'

# 아래는 테스트로 출력해 보기 위한 코드입니다.
print("결과 : {}".format(nextSqure(121)));

File: ProgrammersLearn_COS-PRO2-c/01_Blank/004_grade-counter/question.c
// https://programmers.co.kr/learn/courses/34/lessons/1868

#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int* solution(int scores[], int scores_len) {
    int* grade_counter = (int*)malloc(sizeof(int)*5);
    for(int i = 0; i < 5; ++i)
        grade_counter[i] = 0;

    for(int i = 0; i < scores_len; ++i)
    {
        if( /**/ )
            grade_counter[0] += 1;
        else if( /**/ )
            grade_counter[1] += 1;
        else if( /**/ )
            grade_counter[2] += 1;
        else if( /**/ )
            grade_counter[3] += 1;
        else
            grade_counter[4] += 1;
    }
    return grade_counter;
}
